cron/k8s/transfer.release-v2.yaml

```diff
@@ -34,6 +34,10 @@ spec:
               value: "gs://ossf-scorecard-data-releasetest2"
             - name: SCORECARD_BIGQUERY_TABLE
               value: "scorecard_releasetest2"
+            - name: SCORECARD_COMPLETION_THRESHOLD
+              value: "1"
+            - name: SCORECARD_WEBHOOK_URL
```

Comment:
No need for webhook.

Reply:
Oh, looks like auto-merge was enabled. Could you please remove the webhook? This will cause the docker images to be tagged twice. Although, not an issue, just want to avoid it.

Reply:
let me send a new PR.